### ISSUE TITLE
Failed to start Docker Application Container Engine.

### ISSUE THREAD

**Issue by laithrafid** [docker/docker#27114]
i'm facing this error every time i try to start the docker engine .... 
Failed to start Docker Application Container Engine.

 docker version
Client:
 Version:      1.12.1
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   23cf638
 Built:
 OS/Arch:      linux/amd64

Server:
 Version:      1.12.1
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   23cf638
 Built:
 OS/Arch:      linux/amd64

 docker info
Containers: 0
 Running: 0
 Paused: 0
 Stopped: 0
Images: 6
Server Version: 1.12.1
Storage Driver: devicemapper
 Pool Name: docker-253:1-201329349-pool
 Pool Blocksize: 65.54 kB
 Base Device Size: 10.74 GB
 Backing Filesystem: xfs
 Data file: /dev/loop0
 Metadata file: /dev/loop1
 Data Space Used: 427.8 MB
 Data Space Total: 214.7 GB
 Data Space Available: 48.89 GB
 Metadata Space Used: 1.372 MB
 Metadata Space Total: 2.147 GB
 Metadata Space Available: 2.146 GB
 Thin Pool Minimum Free Space: 21.47 GB
 Udev Sync Supported: true
 Deferred Removal Enabled: false
 Deferred Deletion Enabled: false
 Deferred Deleted Device Count: 0
 Data loop file: /var/lib/docker/devicemapper/devicemapper/data
 WARNING: Usage of loopback devices is strongly discouraged for production use. Use `--storage-opt dm.thinpooldev` to specify a custom block storage device.
 Metadata loop file: /var/lib/docker/devicemapper/devicemapper/metadata
 Library Version: 1.02.107-RHEL7 (2016-06-09)
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: null overlay host bridge
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Security Options: seccomp
Kernel Version: 3.10.0-123.el7.x86_64
Operating System: CentOS Linux 7 (Core)
OSType: linux
Architecture: x86_64
CPUs: 56
Total Memory: 125.6 GiB
Name: localhost.localdomain
ID: B5DS:4QWH:XPGM:NJE7:WXTJ:VQRK:3Q55:XECQ:RNZF:IIOB:MZ4J:Q3ON
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
WARNING: bridge-nf-call-iptables is disabled
WARNING: bridge-nf-call-ip6tables is disabled
Insecure Registries:
 127.0.0.0/8

Journalctl -a Output 

 Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -D OUTPUT -m addrtype --dst-type LOCAL ! --dst 127.0.0.0/8 -j DOCKER' failed: iptables v1.4.21: Couldn't load ta

```
                                                   Try `iptables -h' or 'iptables --help' for more information.
```

Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -D OUTPUT -m addrtype --dst-type LOCAL -j DOCKER' failed: iptables v1.4.21: Couldn't load target `DOCKER':No suc

```
                                                   Try `iptables -h' or 'iptables --help' for more information.
```

Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -D PREROUTING' failed: iptables: Bad rule (does a matching rule exist in that chain?).
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -D OUTPUT' failed: iptables: Bad rule (does a matching rule exist in that chain?).
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -F DOCKER' failed: iptables: No chain/target/match by that name.
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -X DOCKER' failed: iptables: No chain/target/match by that name.
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -F DOCKER' failed: iptables: No chain/target/match by that name.
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -X DOCKER' failed: iptables: No chain/target/match by that name.
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -F DOCKER-ISOLATION' failed: iptables: No chain/target/match by that name.
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -X DOCKER-ISOLATION' failed: iptables: No chain/target/match by that name.
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -n -L DOCKER' failed: iptables: No chain/target/match by that name.
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -n -L DOCKER' failed: iptables: No chain/target/match by that name.
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -n -L DOCKER-ISOLATION' failed: iptables: No chain/target/match by that name.
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -C DOCKER-ISOLATION -j RETURN' failed: iptables: Bad rule (does a matching rule exist in that chain?).
Oct 03 10:42:49 localhost.localdomain NetworkManager[1446]: <info>  (docker0): new Bridge device (carrier: OFF, driver: 'bridge', ifindex: 6)
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -C POSTROUTING -s 172.17.0.0/16 ! -o docker0 -j MASQUERADE' failed: iptables: No chain/target/match by that name
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -C DOCKER -i docker0 -j RETURN' failed: iptables: Bad rule (does a matching rule exist in that chain?).
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -D FORWARD -i docker0 -o docker0 -j DROP' failed: iptables: Bad rule (does a matching rule exist in that chain?).
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -C FORWARD -i docker0 -o docker0 -j ACCEPT' failed: iptables: Bad rule (does a matching rule exist in that ch
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -C FORWARD -i docker0 ! -o docker0 -j ACCEPT' failed: iptables: Bad rule (does a matching rule exist in that
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -C FORWARD -o docker0 -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT' failed: iptables: Bad rule (does
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -C PREROUTING -m addrtype --dst-type LOCAL -j DOCKER' failed: iptables: No chain/target/match by that name.
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -C OUTPUT -m addrtype --dst-type LOCAL -j DOCKER ! --dst 127.0.0.0/8' failed: iptables: No chain/target/match by
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -C FORWARD -o docker0 -j DOCKER' failed: iptables: No chain/target/match by that name.
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -C FORWARD -j DOCKER-ISOLATION' failed: iptables: No chain/target/match by that name.
Oct 03 10:42:49 localhost.localdomain kernel: IPv6: ADDRCONF(NETDEV_UP): docker0: link is not ready
Oct 03 10:42:49 localhost.localdomain avahi-daemon[1365]: Joining mDNS multicast group on interface docker0.IPv4 with address 172.17.0.1.
Oct 03 10:42:49 localhost.localdomain NetworkManager[1446]: <info>  (docker0): device state change: unmanaged -> unavailable (reason 'connection-assumed') [10 20 41]
Oct 03 10:42:49 localhost.localdomain avahi-daemon[1365]: New relevant interface docker0.IPv4 for mDNS.
Oct 03 10:42:49 localhost.localdomain avahi-daemon[1365]: Registering new address record for 172.17.0.1 on docker0.IPv4.
Oct 03 10:42:49 localhost.localdomain NetworkManager[1446]: <info>  ifcfg-rh: add connection in-memory (803caf82-38cd-4ed4-994a-1623fc1f2192,"docker0")
Oct 03 10:42:49 localhost.localdomain NetworkManager[1446]: <info>  (docker0): device state change: unavailable -> disconnected (reason 'connection-assumed') [20 30 41]
Oct 03 10:42:49 localhost.localdomain NetworkManager[1446]: <info>  (docker0): Activation: starting connection 'docker0' (803caf82-38cd-4ed4-994a-1623fc1f2192)
Oct 03 10:42:49 localhost.localdomain NetworkManager[1446]: <info>  (docker0): device state change: disconnected -> prepare (reason 'none') [30 40 0]
Oct 03 10:42:49 localhost.localdomain NetworkManager[1446]: <info>  (docker0): device state change: prepare -> config (reason 'none') [40 50 0]
Oct 03 10:42:49 localhost.localdomain NetworkManager[1446]: <info>  (docker0): device state change: config -> ip-config (reason 'none') [50 70 0]
Oct 03 10:42:49 localhost.localdomain NetworkManager[1446]: <info>  (docker0): device state change: ip-config -> ip-check (reason 'none') [70 80 0]
Oct 03 10:42:49 localhost.localdomain NetworkManager[1446]: <info>  (docker0): device state change: ip-check -> secondaries (reason 'none') [80 90 0]
Oct 03 10:42:49 localhost.localdomain NetworkManager[1446]: <info>  (docker0): device state change: secondaries -> activated (reason 'none') [90 100 0]
Oct 03 10:42:49 localhost.localdomain NetworkManager[1446]: <info>  (docker0): Activation: successful, device activated.
Oct 03 10:42:49 localhost.localdomain dbus-daemon[1368]: dbus[1368]: [system] Activating via systemd: service name='org.freedesktop.nm_dispatcher' unit='dbus-org.freedesktop.nm-dispatcher.service'
Oct 03 10:42:49 localhost.localdomain dbus[1368]: [system] Activating via systemd: service name='org.freedesktop.nm_dispatcher' unit='dbus-org.freedesktop.nm-dispatcher.service'
Oct 03 10:42:49 localhost.localdomain systemd[1]: Starting Network Manager Script Dispatcher Service...
Oct 03 10:42:49 localhost.localdomain dbus[1368]: [system] Successfully activated service 'org.freedesktop.nm_dispatcher'
Oct 03 10:42:49 localhost.localdomain systemd[1]: Started Network Manager Script Dispatcher Service.
Oct 03 10:42:49 localhost.localdomain dbus-daemon[1368]: dbus[1368]: [system] Successfully activated service 'org.freedesktop.nm_dispatcher'
Oct 03 10:42:49 localhost.localdomain nm-dispatcher[3980]: Dispatching action 'up' for docker0
Oct 03 10:42:49 localhost.localdomain systemd[1]: Unit iscsi.service cannot be reloaded because it is inactive.
Oct 03 10:42:49 localhost.localdomain dockerd[3870]: time="2016-10-03T10:42:49.961751785-04:00" level=info msg="Default bridge (docker0) is assigned with an IP address 172.17.0.0/16. Daemon option --bip can be used to set a preferred IP
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -C POSTROUTING -s 172.17.0.0/16 ! -o docker0 -j MASQUERADE' failed: iptables: No chain/target/match by that name
Oct 03 10:42:49 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:49 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -C DOCKER -i docker0 -j RETURN' failed: iptables: Bad rule (does a matching rule exist in that chain?).
Oct 03 10:42:50 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:50 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -D FORWARD -i docker0 -o docker0 -j DROP' failed: iptables: Bad rule (does a matching rule exist in that chain?).
Oct 03 10:42:50 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:50 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -C FORWARD -i docker0 -o docker0 -j ACCEPT' failed: iptables: Bad rule (does a matching rule exist in that ch
Oct 03 10:42:50 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:50 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -C FORWARD -i docker0 ! -o docker0 -j ACCEPT' failed: iptables: Bad rule (does a matching rule exist in that
Oct 03 10:42:50 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:50 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -C FORWARD -o docker0 -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT' failed: iptables: Bad rule (does
Oct 03 10:42:50 localhost.localdomain firewalld[1361]: 2016-10-03 10:42:50 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -C FORWARD -o docker0 -j DOCKER' failed: iptables: No chain/target/match by that name.
Oct 03 10:42:50 localhost.localdomain dockerd[3870]: time="2016-10-03T10:42:50.122470160-04:00" level=info msg="Loading containers: done."
Oct 03 10:42:50 localhost.localdomain dockerd[3870]: time="2016-10-03T10:42:50.122979347-04:00" level=info msg="Daemon has completed initialization"
Oct 03 10:42:50 localhost.localdomain dockerd[3870]: time="2016-10-03T10:42:50.123016497-04:00" level=info msg="Docker daemon" commit=23cf638 graphdriver=devicemapper version=1.12.1

Oct 03 10:42:50 localhost.localdomain dockerd[3870]: time="2016-10-03T10:42:50.122470160-04:00" level=info msg="Loading containers: done."
Oct 03 10:42:50 localhost.localdomain dockerd[3870]: time="2016-10-03T10:42:50.122979347-04:00" level=info msg="Daemon has completed initialization"
Oct 03 10:42:50 localhost.localdomain dockerd[3870]: time="2016-10-03T10:42:50.123016497-04:00" level=info msg="Docker daemon" commit=23cf638 graphdriver=devicemapper version=1.12.1
Oct 03 10:42:50 localhost.localdomain dockerd[3870]: time="2016-10-03T10:42:50.154067444-04:00" level=info msg="API listen on /var/run/docker.sock"
Oct 03 10:42:50 localhost.localdomain systemd[1]: Started Docker Application Container Engine.
Oct 03 10:42:50 localhost.localdomain polkitd[1568]: Unregistered Authentication Agent for unix-process:3865:40165 (system bus name :1.23, object path /org/freedesktop/PolicyKit1/AuthenticationAgent, locale en_US.UTF-8) (disconnected fro
Oct 03 10:43:58 localhost.localdomain polkitd[1568]: Registered Authentication Agent for unix-process:4070:47334 (system bus name :1.26 [/usr/bin/pkttyagent --notify-fd 5 --fallback], object path /org/freedesktop/PolicyKit1/Authenticatio
Oct 03 10:43:58 localhost.localdomain systemd[1]: Stopping Docker Application Container Engine...
Oct 03 10:43:58 localhost.localdomain dockerd[3870]: time="2016-10-03T10:43:58.292909646-04:00" level=info msg="Processing signal 'terminated'"
Oct 03 10:43:58 localhost.localdomain dockerd[3870]: time="2016-10-03T10:43:58.342631223-04:00" level=info msg="stopping containerd after receiving terminated"
Oct 03 10:43:59 localhost.localdomain systemd[1]: Stopped Docker Application Container Engine.
Oct 03 10:43:59 localhost.localdomain polkitd[1568]: Unregistered Authentication Agent for unix-process:4070:47334 (system bus name :1.26, object path /org/freedesktop/PolicyKit1/AuthenticationAgent, locale en_US.UTF-8) (disconnected fro
Oct 03 10:44:04 localhost.localdomain kernel: bio: create slab <bio-0> at 0
Oct 03 10:44:04 localhost.localdomain kernel: bio: create slab <bio-2> at 2
Oct 03 10:44:04 localhost.localdomain systemd[1]: Device dev-disk-by\x2duuid-18afdb40\x2d60e2\x2d4588\x2d9ade\x2d25bf324c57eb.device appeared twice with different sysfs paths /sys/devices/virtual/block/loop0 and /sys/devices/virtual/bloc
Oct 03 10:44:05 localhost.localdomain firewalld[1361]: 2016-10-03 10:44:05 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -D OUTPUT -m addrtype --dst-type LOCAL -j DOCKER' failed: iptables: No chain/target/match by that name.
Oct 03 10:44:05 localhost.localdomain firewalld[1361]: 2016-10-03 10:44:05 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -D PREROUTING' failed: iptables: Bad rule (does a matching rule exist in that chain?).
Oct 03 10:44:05 localhost.localdomain firewalld[1361]: 2016-10-03 10:44:05 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -D OUTPUT' failed: iptables: Bad rule (does a matching rule exist in that chain?).
Oct 03 10:44:05 localhost.localdomain firewalld[1361]: 2016-10-03 10:44:05 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -X DOCKER' failed: iptables: Too many links.
Oct 03 10:44:05 localhost.localdomain firewalld[1361]: 2016-10-03 10:44:05 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -X DOCKER-ISOLATION' failed: iptables: Too many links.
Oct 03 10:44:05 localhost.localdomain firewalld[1361]: 2016-10-03 10:44:05 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -n -L DOCKER' failed: iptables: No chain/target/match by that name.
Oct 03 10:44:05 localhost.localdomain firewalld[1361]: 2016-10-03 10:44:05 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -C DOCKER-ISOLATION -j RETURN' failed: iptables: Bad rule (does a matching rule exist in that chain?).
Oct 03 10:44:05 localhost.localdomain firewalld[1361]: 2016-10-03 10:44:05 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -C DOCKER -i docker0 -j RETURN' failed: iptables: Bad rule (does a matching rule exist in that chain?).
Oct 03 10:44:05 localhost.localdomain firewalld[1361]: 2016-10-03 10:44:05 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -D FORWARD -i docker0 -o docker0 -j DROP' failed: iptables: Bad rule (does a matching rule exist in that chain?).
Oct 03 10:44:05 localhost.localdomain firewalld[1361]: 2016-10-03 10:44:05 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -C PREROUTING -m addrtype --dst-type LOCAL -j DOCKER' failed: iptables: No chain/target/match by that name.
Oct 03 10:44:05 localhost.localdomain firewalld[1361]: 2016-10-03 10:44:05 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -C OUTPUT -m addrtype --dst-type LOCAL -j DOCKER ! --dst 127.0.0.0/8' failed: iptables: No chain/target/match by
Oct 03 10:44:05 localhost.localdomain firewalld[1361]: 2016-10-03 10:44:05 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -C POSTROUTING -s 172.17.0.0/16 ! -o docker0 -j MASQUERADE' failed: iptables: No chain/target/match by that name
Oct 03 10:44:05 localhost.localdomain firewalld[1361]: 2016-10-03 10:44:05 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t nat -C DOCKER -i docker0 -j RETURN' failed: iptables: Bad rule (does a matching rule exist in that chain?).
Oct 03 10:44:05 localhost.localdomain firewalld[1361]: 2016-10-03 10:44:05 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -D FORWARD -i docker0 -o docker0 -j DROP' failed: iptables: Bad rule (does a matching rule exist in that chain?).
Oct 03 10:44:05 localhost.localdomain firewalld[1361]: 2016-10-03 10:44:05 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -C FORWARD -i docker0 -o docker0 -j ACCEPT' failed: iptables: Bad rule (does a matching rule exist in that ch
Oct 03 10:44:05 localhost.localdomain firewalld[1361]: 2016-10-03 10:44:05 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -C FORWARD -i docker0 ! -o docker0 -j ACCEPT' failed: iptables: Bad rule (does a matching rule exist in that
Oct 03 10:44:05 localhost.localdomain firewalld[1361]: 2016-10-03 10:44:05 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -C FORWARD -o docker0 -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT' failed: iptables: Bad rule (does
Oct 03 10:44:05 localhost.localdomain firewalld[1361]: 2016-10-03 10:44:05 ERROR: COMMAND_FAILED: '/sbin/iptables -w2 -t filter -C FORWARD -o docker0 -j DOCKER' failed: iptables: No chain/target/match by that name.
Oct 03 10:45:20 localhost.localdomain polkitd[1568]: Registered Authentication Agent for unix-process:4241:55584 (system bus name :1.28 [/usr/bin/pkttyagent --notify-fd 5 --fallback], object path /org/freedesktop/PolicyKit1/Authenticatio
Oct 03 10:45:20 localhost.localdomain systemd[1]: Starting Docker Application Container Engine...
Oct 03 10:45:20 localhost.localdomain dockerd[4246]: time="2016-10-03T10:45:20.831516172-04:00" level=fatal msg="Error starting daemon: pid file found, ensure docker is not running or delete /var/run/docker.pid"
Oct 03 10:45:20 localhost.localdomain systemd[1]: docker.service: main process exited, code=exited, status=1/FAILURE
Oct 03 10:45:20 localhost.localdomain systemd[1]: Failed to start Docker Application Container Engine.
Oct 03 10:45:20 localhost.localdomain systemd[1]: Unit docker.service entered failed state.
Oct 03 10:45:20 localhost.localdomain systemd[1]: docker.service failed.

**Comment by laithrafid** [comment]
Uname Linux localhost.localdomain 3.10.0-123.el7.x86_64 #1 SMP Mon Jun 30 12:09:22 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux

**Comment by mrjana** [comment]
Even though `firewalld` is reporting a lot of non-existing chain errors they won't stop daemon from starting. The actual error for daemon startup failure is this:
```
04:00" level=fatal msg="Error starting daemon: pid file found, ensure docker is not running or delete /var/run/docker.pid"
```

In fact if you are trying to run multiple daemons like this in parallel with iptables enabled then this failure as well as all the iptables errors are bound to happen.

**Comment by laithrafid** [comment]
so docker does need tables ?

**Comment by mrjana** [comment]
Closing this as a case of starting two daemons at the same time in same exec root and one of them failing to start. Please feel free to reopen if you think otherwise.
